client/app/api/course/Statistics/CourseStatistics.js

```diff
@@ -42,6 +42,6 @@ export default class CourseStatisticsAPI extends BaseCourseAPI {
   }
 
   get #urlPrefix() {
-    return `/courses/${this.courseId}/statistics/course`;
+    return `/courses/${this.courseId}/statistics`;
```

Comment:
Can you rewrite this into ts and properly add the type for the api response.

Reply:
done